modules/apps_api/app/swagger/apps_api/v0/apps.yml

```diff
@@ -54,14 +49,42 @@ paths:
                     type: array
                     items:
                       $ref: "#/components/schemas/App"
-                  meta:
-                    type: object
-                    properties:
-                      count:
-                        type: number
-                        description: 'Counter used for pagination'
-                        example: 1
-
+                  #meta:
+                    #type: object
+                    #properties:
+                      #count:
+                        #type: number
+                        #description: 'Counter used for pagination'
+                        #example: 1
```

Comment:
does this need to stick around or can we just remove it?

Reply:
commented out since it'll be implemented in the coming weeks